### PR TITLE
Align project config

### PR DESCRIPTION
**Pull request by felixfontein**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -38,6 +38,44 @@ classifiers = [
...
+dev = [
+    # Used by nox sessions
+    "antsibull-docs-parser[codeqa]",
+    "antsibull-docs-parser[coverage]",
+    "antsibull-docs-parser[formatting]",
+    "antsibull-docs-parser[test]",
+    "antsibull-docs-parser[typing]",
+    # formatters
+    "isort",
+    "black",
```
Comment (gotmax23) [inline pyproject.toml:74]:
```suggestion
```

File: pyproject.toml [hunk 2]
```diff
@@ -38,6 +38,44 @@ classifiers = [
...
+]
+formatters = [
+    "black",
+    "isort",
+]
+dev = [
+    # Used by nox sessions
+    "antsibull-docs-parser[codeqa]",
+    "antsibull-docs-parser[coverage]",
+    "antsibull-docs-parser[formatting]",
```
Comment (gotmax23) [inline pyproject.toml:69]:
```suggestion
    "antsibull-docs-parser[formatters]",
```

File: pyproject.toml [hunk 3]
```diff
@@ -38,6 +38,41 @@ classifiers = [
 [project.urls]
 "Source code" = "https://github.com/ansible-community/antsibull-docs-parser/"
 
+[project.optional-dependencies]
```
[the remark this answers is not in the document]
Comment (felixfontein) [reply pyproject.toml:41]: I did order them, *except* `dev`, which I think is best at the end since it depends on some of the previous sections.

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing this!

**Pull request merged**